src/modm/platform/core/rp/module.md

```diff
@@ -0,0 +1,102 @@
+# RP2040 core module
+
+This module specializes the generic `modm:platform:cortex-m` module with
+RP2040-specific startup code and linkerscripts.
+
+!!! note "Second-stage Bootloader"
+	The external Flash memory requires additional setup, which is provided by
+	the second-stage bootloader called boot2. You can select a pre-defined
+	configuration via the `modm:platform:core:boot2` option.
+
+
+## Startup
+
+The `__modm_initialize_platform()` callback resets all hardware to a known
+initial state.
+
+
+## Linkerscript
+
+The RP2040 currently on has one linkerscript template for internal SRAM and
```

Comment:
"The RP2040 currently on has one linkerscript template", something is not right there.